### PR TITLE
Updated the get token part

### LINKED ISSUE #4870: [Docs] User Tokens: What they are and how to get yours
**Current State:**
The Meshery REST API requires a valid user token for most of its endpoints, but not all.

**Desired State:**
Meshery docs should describe how users can download their token.

This page can be updated - https://docs.meshery.io/extensibility/api

---
#### [Contributor Guides](https://docs.meshery.io/project/contributing) and Resources
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by unnati914**
**Description**

This PR fixes #4870 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit fb4f45d7e3b450ac836bc2160eb8e4a588238055 at: https://61ee61e14c3e407032bac9d3--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit c5842d105eae53c23b32114da90f15f0b31c0df8 at: https://61ee6455c5ec596a09828fac--meshery-docs-preview.netlify.app

File: docs/pages/extensibility/api.md [hunk 1]
```diff
@@ -25,6 +25,29 @@ Each of the API endpoints are exposed through [server.go](https://github.com/lay
...
+Tokens allow a user to authenticate with cloud apps and bypass two step verification and SSO, retrieve data from the instance through REST APIS
+An authentication token (security token) is a “trusted device” used to access an electronically restricted resource (usually an application or a corporate network). It can be seen as an electronic key that enables a user to authenticate and prove his identity by storing some sort of personal information.
+
+
+- Firstly, lets discuss how to get the token using Meshery-UI
+1. Login/Signup to the dashboard of Meshery-UI
+2. Click on Get token from the dropdown of profile section, 
+3. You can download your token from there
+
+- Secondly, you can download your token using Meshery-CLI
```
Comment (bariqhibat) [inline docs/pages/extensibility/api.md:44]: Will it be better to phrase it as `mesheryctl` or Meshery-CLI?

File: docs/pages/extensibility/api.md [hunk 2]
```diff
@@ -25,6 +25,29 @@ Each of the API endpoints are exposed through [server.go](https://github.com/lay
...
 Alternatively, [Remote Providers](./providers) can extend Meshery's endpoints behind the `/api/extensions/` endpoint.
 
+### How to get your token
+
+There are two ways to get your token:
+1. Meshery UI
+2. Meshery CLI
+
+- What are tokens?
+Tokens allow a user to authenticate with cloud apps and bypass two step verification and SSO, retrieve data from the instance through REST APIS
```
Comment (bariqhibat) [inline docs/pages/extensibility/api.md:35]: nitpick: `REST APIs`

**Review by bariqhibat — COMMENTED** [review]
Some comments, thanks for the contribution @unnati914

**Comment by unnati914** [comment]
cool thank you so much!

File: docs/pages/extensibility/api.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/extensibility/api.md:44]: If it was "Meshery-CLI", which to @bariqhibat's point it probably isn't, you'll want to drop the hyphenation and use the indefinite article in reference to it - there is only one Meshery CLI.

**Comment by leecalcote** [comment]
@unnati914, good. 👍 Your PR / issue titles are coming along. :) Will you prepend this one with a `[Docs]`?

File: docs/pages/extensibility/api.md (hunk 2 above)
Comment (leecalcote) [reply docs/pages/extensibility/api.md:35]: and GraphQL APIs, too. It would be good to include a link to both of these here.

**Comment by unnati914** [comment]
hey @leecalcote :) sorry, I didn't notice it thank you for merging it, will take care next time

**Pull request merged**